### PR TITLE
Add unzip function

### PR DESCRIPTION
**Pull request by AlienKevin**
#18

### CODE REVIEW

File: src/Array/Extra.elm [hunk 1]
```diff
@@ -233,6 +233,19 @@ zip3 =
     map3 (\a b c -> ( a, b, c ))
 
 
+{-| Unzip array of tuples into a tuple containing two arrays
+    unzip [ (1, 'a'), (2, 'b'), (3, 'c') ] == ([ 1, 2, 3 ], [ 'a', 'b', 'c' ])
```
Comment (jonboiser) [inline src/Array/Extra.elm:237]: Could you add a newline between the description and example?

Also, when trying this out, I had to coerce the list-literal to an Array using `Array.fromList` like this

```
unzip <| fromList [ (1, 'a'), (2, 'b'), (3, 'c') ]
```

I'm not sure, if this is a recent change in Elm, as all the other examples in that file assume that you can write array literals using square brackets.

In any case, could you update the example to convert the List to Array just so the example works as typed?

Comment (AlienKevin) [reply src/Array/Extra.elm]: I fixed the issues you mentioned.

**Comment by jonboiser** [comment]
@prikhi Would you mind making a new release with this PR?

I'm happy to help out with releases too, I just haven't done it in several years. Is there documentation that shows how to do it?

**Comment by prikhi** [comment]
I think last time I did something like this:
```
elm bump
git commit -a -m'Prepare for vX.Y.Z'
git tag X.Y.Z # add the -a flag if you want to add a description to the tag
git push --follow-tags
elm publish
```

**Comment by jonboiser** [comment]
Okay I can try that. Do I need to get an API key or something to publish? I probably did this before, but forgot.

**Comment by prikhi** [comment]
Nope

**Comment by jonboiser** [comment]
I was able to publish 2.3.0 adding `unzip`, thanks @prikhi and @AlienKevin !

**Pull request merged**